### PR TITLE
Add banner expiry into localstorage and its functional now

### LINKED ISSUE #1457: Maintained collections display persistent "maintenance request approved" message
**Describe the bug**
In the particular build produced by #1449, whenever I view a collection record that I am maintainer of, I get the "maintaining request approved" banner, even though I claimed the record weeks ago

**To Reproduce**
1. Log on as allysonlister
2. visit https://deploy-preview-1449--fairsharing.netlify.app/3519
3. See error

**Expected behavior**
Banner should only be there for a predefined time after approval.

**Screenshots**
maintain

**Operating system and browser version**
Ubuntu / Firefox

**Additional context**
Does not happen on production site.

### PR DESCRIPTION
**Pull request by hoseinmirian**
failed or success banner in record display now remains 2 days and then disappears. I am confident but you can give it 2 days to see if that works and then approve the branch. tnx

in case you wanted to actually see the result I can make a commit and change it to 1min to get expired and then revert back changes in another commit.

### CODE REVIEW

**Comment by hoseinmirian** [comment]
failed or success banner in record display now remains 2 days and then disappears. I am confident but you can give it 2 days to see if that works and then approve the branch. tnx

File: src/views/Records/Record.vue [hunk 1]
```diff
@@ -653,10 +653,8 @@ export default {
               const expiryDate = new Date(temp[_module.getField("id")]);
               let now = new Date();
               const DAY = 2;
-              const isExpired = expiryDate.getTime() + (DAY * 24 * 60 * 60 * 1000) < now.getTime();
-              if (isExpired) {
-                _module.showBanner = false;
-              }
+              // very important line: instead of adding if its been expired I directly assigned to variable so test can be passed much easier.
+              _module.showBanner = !(expiryDate.getTime() + (DAY * 24 * 60 * 60 * 1000) < now.getTime());
```
Comment (hoseinmirian) [inline src/views/Records/Record.vue:657]: day * hour * min * sec * 1000 = time as milisecond.

**Comment by allysonlister** [comment]
Please could you re-commit to 1 minute for now, as you suggest? 2 days from now is the weekend and I don't think I'll check then :-D thanks!

**Comment by hoseinmirian** [comment]
please visit a new record and ask for maintenance.(a record u never ask for maintenance):
1- approve it from the curator panel and give it 1 min and then refresh
2-  reject and do give it 1 min and refresh

**Review by allysonlister — CHANGES_REQUESTED** [review]
Thank you, the green "your request has been approved" is working as expected, as the red "rejected" banner. 

However, the initial message being displayed after the user requests ownership isn't the "your request has been submitted", but rather states that "you have already submitted a request". See gif below.
already

**Review by knirirr — APPROVED** [review]
This looks OK to me, though I agree with @allysonlister that different text would be better.

**Comment by hoseinmirian** [comment]
@allysonlister if I replace the wording with urs is all fine? 

Im asking because I want to revert back changes in the next commit to 2days.

**Comment by allysonlister** [comment]
Thanks!

@hoseinmirian - just to be explicit, here is some text you can use instead:

"Your ownership request for this record has been submitted. We will get back to you within a few working days."

**Review by allysonlister — APPROVED** [review]
Thank you!

**Pull request merged**